runner/aos-service@.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=AOS Service
+After=network.target
+StartLimitIntervalSec=30
+StartLimitBurst=3
+
+[Service]
+Type=forking
+Restart=always
+RestartSec=1
+ExecStartPre=/usr/bin/{RUNNER} delete -f %i
+ExecStart=/usrt/bin/{RUNNER} run -d --pid-file /var/aos/runtime/%i/.pid -b /var/aos/runtime/%i %i
```

Comment:
In all  cases /var/aos/runtime -> /run/aos/runtime

Reply:
fixed